packages/core/src/index.ts

```diff
@@ -1,6 +1,9 @@
 export {
   setupI18n,
   I18n,
+} from "./i18n"
+
+export type {
   AllMessages,
   MessageDescriptor,
   Messages,
```

Comment:
Without typescript plugin, rollup treat type only exports as regular, and obviously fail. Here we do a little help for him and manually mark them as type only.